roles/prometheus/defaults/main.yml

```diff
@@ -0,0 +1,81 @@
+---
+# Kubernetes configs path
+k8s_conf_dir: /etc/kubernetes
+k8s_prometheus_dir: '{{ k8s_conf_dir }}/addons/prometheus'
+
+# Master hosts names
+k8s_master_hosts: {}
+
+# Retention days (how long to store data)
+k8s_prometheus_retention_days: 7
```

Comment:
We should add this parameter with comments in `group_vars/all.yml` also

Reply:
ok